content/en/docs/setup/quickstart.md

```diff
@@ -8,100 +8,275 @@ menu:
+  labels:
+    app: sleep
+    service: sleep
+spec:
+  ports:
+  - port: 80
+    name: http
+  selector:
+    app: sleep
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sleep
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sleep
+  template:
+    metadata:
+      labels:
+        app: sleep
+    spec:
+      terminationGracePeriodSeconds: 0
+      serviceAccountName: sleep
+      containers:
+      - name: sleep
+        image: curlimages/curl
+        command: ["/bin/sleep", "infinity"]
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - mountPath: /etc/sleep/tls
+          name: secret-volume
+      volumes:
+      - name: secret-volume
+        secret:
+          secretName: sleep-secret
+          optional: true
+EOF
+```
+
+```console
+kubectl apply -f -<<
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: httpbin
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: httpbin
+  labels:
+    app: httpbin
+    service: httpbin
+spec:
+  ports:
+  - name: http
+    port: 8000
+    targetPort: 80
+  selector:
+    app: httpbin
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: httpbin
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: httpbin
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: httpbin
+        version: v1
+    spec:
+      serviceAccountName: httpbin
+      containers:
+      - image: docker.io/kong/httpbin
+        imagePullPolicy: IfNotPresent
+        name: httpbin
+        ports:
+        - containerPort: 80
+EOF
+```
 
-  ```sh
-  # get kmesh.yaml from deploy/yaml/kmesh.yaml
-  [root@ ~]# kubectl apply -f kmesh.yaml
-  [root@ ~]# kubectl apply -f clusterrole.yaml
-  [root@ ~]# kubectl apply -f clusterrolebinding.yaml
-  [root@ ~]# kubectl apply -f serviceaccount.yaml
-  ```
+Check sample applications status:
 
-  By default, the Kmesh base function is used, other function can be selected by adjusting the startup parameters in the yaml file.
+```console
+kubectl get pod 
+NAME                                      READY   STATUS    RESTARTS   AGE
+httpbin-65975d4c6f-96kgw                  1/1     Running   0          3h38m
+sleep-7656cf8794-8tp9n                    1/1     Running   0          3h38m
+```
 
-- Check kmesh service status
+You can determine if a pod is managed by kmesh by looking at the pod's annotation.
 
-  ```sh
-  [root@ ~]# kubectl get pods -A | grep kmesh
-  kmesh-system   kmesh-l5z2j                                 1/1     Running   0          117m
-  ```
+```console
+kubectl describe po httpbin-65975d4c6f-96kgw
+Name:             httpbin-65975d4c6f-96kgw
+Namespace:        default
+Priority:         0
+Service Account:  httpbin
+Node:             kmesh-control-plane/172.18.0.2
+Start Time:       Fri, 26 Apr 2024 11:54:03 +0800
+Labels:           app=httpbin
+                  pod-template-hash=65975d4c6f
+                  version=v1
+Annotations:      kmesh.net/redirection: enabled
```

Comment:
Just grep to check if it has `kmesh.net/redirection` annotations is enough, make page laconic